rules/manager.go

```diff
@@ -381,6 +398,7 @@ func (g *Group) Eval(ctx context.Context, ts time.Time) {
 
 			evalTotal.Inc()
 
+			rule.SetEvaluationTimestamp(time.Now())
```

Comment:
Same as for groups, should the rule eval timestamp only be set after a successful evaluation, together with the duration (in the `defer` above?).

Reply:
(or not necessarily *successful* duration)